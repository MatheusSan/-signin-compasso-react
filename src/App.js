import React from "react";
import GlobalStyles from "./GlobalStyle";
import Login from "./pages/login/Login";

export default function App() {
  return (
    <>
      <GlobalStyles />
      <Login />
    </>
  );
}
