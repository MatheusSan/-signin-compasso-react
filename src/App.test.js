import React from "react";
import { render, screen } from "@testing-library/react";
import "@testing-library/jest-dom";

import App from "./App";

describe("Componente principal", () => {
  it("Mostrar o login",  () => {
    render(<App/>);
    expect(screen.getByText("Continuar")).toBeInTheDocument();
  });
  it("O componente App deve ser sempre o mesmo", () => {
    const { container } = render(<App/>);
    expect(container.firstChild).toMatchSnapshot();
  });
});





