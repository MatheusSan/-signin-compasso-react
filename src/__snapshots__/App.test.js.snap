// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Componente principal O componente App deve ser sempre o mesmo 1`] = `
<div
  class="sc-dkPtRN bMWDLf"
>
  <p
    class="sc-iqseJM enrbZi"
  />
  <div
    class="sc-hKwDye xYJTH"
  >
    <form
      class="sc-eCImPb jUxdjF"
    >
      <p
        class="sc-gKclnd dMSGEC"
      >
        Olá,
      </p>
      <p
        class="sc-iCfMLu bTexRS"
      >
        Para continuar navegando de forma
        <br />
         segura, efetue o login na rede.
      </p>
      <p
        class="sc-furwcr iogRvS"
      >
        Login
      </p>
      <input
        class="sc-gsDKAQ iaOfeA"
        name="email"
        placeholder="Usuário"
        type="email"
      />
      <p
        class="sc-pVTFL eIafhs"
      />
      <input
        class="sc-gsDKAQ iaOfeA"
        name="password"
        placeholder="Senha"
        type="password"
      />
      <p
        class="sc-jrQzAO fBjQOU"
      />
      <button
        class="sc-bdvvtL AEFUq"
        type="submit"
      >
        Continuar
      </button>
    </form>
  </div>
  <div
    class="sc-jRQBWg gTPgae"
  >
    <p
      class="sc-kDTinF ifFJdD"
    />
  </div>
</div>
`;
